secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackCookie.scala

```diff
@@ -55,8 +51,6 @@ case class CookieSchema(dataVersion: DataVersion,
                         columnIdMap: Map[UserColumnId, ColumnId],
                         strategyMap: Map[UserColumnId, ComputationStrategyInfo],
                         obfuscationKey: Array[Byte],
-                        computationRetriesLeft: Int,
```

Comment:
is this case class also what's used for parsing/writing to the cookie field in the secondary_manifest? (meaning: does removing this mean we'll stop writing it, and will we still read cookies that have those fields ok, just ignore them?)

Reply:
Yes and yes, respectively - it's standard AutomaticJsonCodecBuilder thing.